src/test/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/documents/recallnotification/RecallNotificationServiceTest.kt

```diff
@@ -86,6 +88,54 @@ internal class RecallNotificationServiceTest {
     )
   }
 
```

Comment:
Can we improve the name of the test before this one?  Currently it reads:
"get recall notification returns recall summary, revocation order and letter to probation when one doesnt exist already".
That doesn't tell me what "one" refers to.  And it's difficult to see how it compares to the following test.